docker/README.md

```diff
@@ -8,9 +8,9 @@ These builds of LightGBM all train on the CPU. For GPU-enabled builds, see [the
 
 Follow the general installation instructions [on the Docker site](https://docs.docker.com/install/):
 
-* [macOS](https://docs.docker.com/docker-for-mac/install/)
-* [Ubuntu](https://docs.docker.com/install/linux/docker-ce/ubuntu/)
-* [Windows](https://docs.docker.com/docker-for-windows/install/)
+- [macOS](https://docs.docker.com/docker-for-mac/install/)
+- [Ubuntu](https://docs.docker.com/install/linux/docker-ce/ubuntu/)
+- [Windows](https://docs.docker.com/docker-for-windows/install/)
```

Comment:
```suggestion
* [macOS](https://docs.docker.com/docker-for-mac/install/)
* [Ubuntu](https://docs.docker.com/install/linux/docker-ce/ubuntu/)
* [Windows](https://docs.docker.com/docker-for-windows/install/)
```

Was this particular change actually the result of the `typo` pre-commit hook somehow? Or did your local editor settings do this?

Unless this is necessary to satisfy any of the project's linters, or unless it improves the rendering somehow, could you please revert them?

Reply:
That was my local editor, reverted!

Reply:
No problem, thanks.

> *reverted!*

Did you forget to push a commit? This is still showing up in the diff.